docs/TESTING.md

```diff
@@ -1,53 +1,60 @@
 ### Test instructions (for developers):
 
-This is a rough sketch, some more background is found in [JM wiki](https://github.com/Joinmarket-Org/joinmarket/wiki/Testing)
+Work in your `jmvenv` virtualenv as for all Joinmarket work. Make sure to have [bitcoind](https://bitcoin.org/en/full-node) 0.17 or newer installed. Also need miniircd installed to the root (i.e. in your `joinmarket-clientserver` directory):
 
-Make sure to have [bitcoind](https://bitcoin.org/en/full-node) 0.17 or newer installed. Also need miniircd installed to the root (i.e. in your `joinmarket-clientserver` directory):
+    (jmvenv)$ cd /path/to/joinmarket-clientserver
+    (jmvenv)$ git clone https://github.com/Joinmarket-Org/miniircd
 
-    cd ~/joinmarket-clientserver
-    git clone https://github.com/Joinmarket-Org/miniircd
+Install the test requirements:
 
-Install the test requirements (still in your virtualenv as mentioned above):
+    (jmvenv)$ pip install -r requirements/testing.txt
 
-    pip install -r requirements/testing.txt
+#### Running the test suite.
 
-Running the test suite should be done something like (advisable to wipe ~/.bitcoin/regtest first):
+Have a `bitcoin.conf` ready in some location, whose contents only need to be:
 
-    pytest --btcconf=/path/to/bitcoin.conf --btcroot=/path/to/bitcoin/bin/ --btcpwd=whatever --nirc=2 --ignore test/test_full_coinjoin.py -p no:warnings
-    
-(you'll first want to copy the regtest_joinmarket.cfg file from the test/ directory to the root directory,
-this file will need minor edits for your btc configuration).
+```
+rpcuser=bitcoinrpc
+rpcpassword=123456abcdef
+fallbackfee=0.0002
+```
 
-### Running tests of sendpayment and tumbler (including with malicious makers)
+(any random password is fine of course). It is also advisable to wipe ~/.bitcoin/regtest first, in case it gets large and slow to process.
 
-(As well as the below, there is a rudimentary "single, automatic, end-to-end coinjoin test" in the file test/test_full_coinjoin.py. Recommended to run it
-with -s to see log output; what's "rudimentary" here is that errors may simply result in the process hanging, so you'll want to investigate in that case.
-Use command line:
+Then copy the `regtest_joinmarket.cfg` file from the `test/` directory to the root directory, you probably won't need to change anything here except perhaps the above password, and the `native` setting if you're doing bech32 wallet tests.
 
-    pytest --btcconf=/path/to/bitcoin.conf --btcroot=/path/to/bitcoin/bin/ --btcpwd=whatever --nirc=2 test/test_full_coinjoin.py -s
+Run the test suite via pytest:
 
-)
+    (jmvenv)$ pytest --btcconf=/path/to/bitcoin.conf --btcroot=/path/to/bitcoin/bin/ --btcpwd=whatever --nirc=2 --ignore test/test_full_coinjoin.py -p no:warnings
```

Comment:
Nit: for sake of consistency, change to `--btcpwd=123456abcdef`?

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/commit/29875b4a2ae151afbc3223271c8c7d5c325f0e46